client/index.html

```diff
@@ -5,9 +5,98 @@
+    <!-- Import Allerta fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Allerta&display=swap" rel="stylesheet">
+    <!-- Add icon library -->
+    <script src="https://kit.fontawesome.com/490bacaf0b.js" crossorigin="anonymous"></script>
+    <link rel="icon" href="./images/grabbit-icon.jpg" />
+    <title>Cohabit</title>
 </head>
 <body>
-    
+    <header id="top">
+        <section class="top-bar">
+            <button class="dark-mode-btn">Switch to dark mode</button>
+            <nav class="nav-bar">            
+                <a class="active" id="register"><i class="fas fa-user-plus"></i></a>
+                <a id="login"><i class="fas fa-sign-in-alt"></i></a>
+            </nav>
+        </section>
+
+
+        <div class="heading">
+            <div id="title-icon">
+                <i class="fas fa-fist-raised" id="title-icon"></i>
+            </div>
+            <div id="title">
+                <h1 id="app-name">grabbit</h1>
+                <h5 id="tagline">habbit your way</h3>
+            </div>            
+        </div>
+    </header>
+
+    <main>
+        <div class="auth-btns">
+            <button class="login-btn">login</button>
+            <button class="register-btn">register</button>            
+        </div>
+
+        <form class="auth-form">
+            <label for="username">username</label>
+            <input type="text" id="username" name="username" placeholder="Enter your username"><br><br>
+            <label for="password">password</label>
+            <input type="text" id="password" name="password" placeholder="Enter your password"><br><br>
+        </form>
+        <!-- <p>Top</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Whitespace</p>
+        <p>Bottom</p>
+        <p>Bottom</p> -->
+    </main>
+    <footer>
+        <div class="bottom-nav" id="bottom-nav-bar">
+            <a href="#bottomMenu" class="active">More <i class="fas fa-arrow-circle-right"></i></a>
+            <a href="https://github.com/natbibi/co-habiters.git">About</a>
```

Comment:
i was thinking it might be better to hide these within a tab on the fixed bottom nav bar / or have it hidden until scroll (i think Rishi suggested this?) because for UI/UX purposes the main options should be on show first like "add a habit", "see all habits". So maybe we can have along the nav "add a habit", "see all habits", "settings" and within settings we can have about/contact/support option.  maybe we can chuck darkmode in there too

Reply:
Sorry, I know it doesn't look like it, but if you read this alongside the navStyles.css and navFunction.js files, I've arranged it so that on a mobile device, the bottom nav is fixed to bottom and appears as a burger bar icon, which you can then tap on and then the other items display vertically (might be easier if you pull down and try it on your machine to see what I mean).  Does that fit our intended brief?

Reply:
Should set up a client branch to merge into and then its easier for us to look on local machine for future stuff, without disrupting server side/ staging and main.

I just clocked that maybe if we keep the burger bar as standard it could be more mobile friendly. Plus if were doing mobile first approach then we dont really need it to breakpoint downwards, cos it will just be zoomed in on larger desktop screen. So for example we make everything max width around 500px (and/or contained within body div with max width), add a breakpoint for 800px ish for larger screens (ipad) to increase font sizes and then it should look like that for desktop too.